src/util/Params.cpp

```diff
@@ -821,6 +821,9 @@ Params::Params() : finalized_(false) {
     "neutral loss peaks have an intensity 1/10 of the primary peak. Neutral losses "
     "are not yet implemented for the res-ev score function.",
     "Available for tide-search.", true);
+  InitIntParam("min-precursor-charge", 1, 1, BILLION,
```

Comment:
It looks like you added the "min-precursor-charge" parameter but did not remove "spectrum-charge" parameter.

Reply:
The spectrum-charge option is still used by other programs in CRUX.